content/messaging/agent-chat-api/v3.4/rtm-reference/index.mdx

```diff
@@ -3756,6 +3757,28 @@ Informs that a chat was deactivated by closing the currently open thread.
 | --------- | --------------------------------------------- |
 | `user_id` | Missing if a thread was closed by the router. |
 
+### `chat_deleted`
+
+Informs that a chat was deleted.
+
+<CodeResponse title={'Sample push payload'}>
+
+```json
+{
+  "chat_id": "PJ0MRSHTDG",
+  "thread_id": "K600PKZON8"
```

Comment:
Should there be `thread_id` in this push?

Reply:
nope - removing